docs/terms-concepts/network-topologies.md

```diff
@@ -5,46 +5,29 @@ title: Network topologies
+In a distributed authority topology, game instances share responsibility for owning and tracking the state of objects in the network using the [Distributed authority model](authority.md#distributed-authority). A small, lightweight central state service tracks a minimal amount of game state and routes network traffic. There is no server simulating the game: each connected game instance runs a subdivision of the simulation.
 
-Distributed authority topologies are useful for keeping costs down, and solve a lot of the input-related issues typically addressed using client prediction systems, although the lack of a central authority can make them more vulnerable to cheating and bad actors.
+Distributed authority topologies are useful for keeping costs down, and solve a lot of the input-related issues typically addressed using client prediction systems. The lack of a central authority can make them more vulnerable to cheating and bad actors.
 
 For more details about distributed authority topologies, refer to the [Distributed authority page](distributed-authority.md).
 
-## Other network topologies
-
-### Local or couch multiplayer
-
-Local multiplayer games use a single client runtime instance that can be played by two or more people on the same screen in the same physical location.
-
-### Offline local area network (LAN)
-
-Local area network games aren't connected to the internet, but use a hosted server onsite to support a local network of clients in the same physical location.
-
-### Peer-to-peer (P2P)
+## Client-server versus distributed authority topology
 
-A peer-to-peer network is created when two or more computers are connected and share resources without going through a separate server.
+The primary differences between [client-server](client-server.md) and [distributed authority](distributed-authority.md) topologies are latency, cost, and security.
 
-![Peer-to-Peer](/img/peer2peer1.png)
+* **Latency**: Client-server topologies typically experience more latency, since all changes must be communicated to a relay service, then the server (where updates are authorized), then back out to clients, whereas clients in distributed authority networks have the authority to communicate changes directly to other clients via the relay service.
+* **Cost**: Client-server topologies using a dedicated game server are typically an expensive option due to the hardware and running costs involved in maintaining a powerful server machine, although this does allow for higher performance and better [tick rates](../learn/ticks-and-update-rates.md). Distributed authority networks are cheaper, since they lack a dedicated central server and game simulations run solely on clients.
+* **Security**: Because client-server topologies have a single central authority, they're more resilient to cheating and bad actors, and can support additional features such as client prediction and rollback.
```

Comment:
Perhaps something like this?

> **Security**: A client-server network topology using a dedicated server is more resilient to cheating and bad actors because it is a single central authority instance. A client-server network topology using a listen server is actually less secure (over-all) than a distributed authority network topology because a bad actor can be in control of the listen server. A distributed authority network topology spreads the authority across multiple clients which reduces the foot print for bad actors down to only the objects their client instance has authority over.